File: app/src/main/java/com/animation/master/dashboard/MainViewModel.java
package com.animation.master.dashboard;

import android.content.Context;
import android.view.View;
import com.animation.master.AppNavigator;

public class MainViewModel {

    private Context mContext;

    public MainViewModel(Context context) {
        this.mContext = context;
    }

    public void onTranslationScaleActivityClick(View view) {
        AppNavigator.navigateToTranslationScaleActivity(mContext);
    }

}

File: app/src/main/java/com/animation/master/dashboard/MainViewModel.kt
package com.animation.master.dashboard

import android.content.Context
import android.view.View
import com.animation.master.AppNavigator

class MainViewModel(private val mContext: Context) {

    fun onTranslationScaleActivityClick(view: View) {
        AppNavigator.navigateToTranslationScaleActivity(mContext)
    }

}
